readme-template.hbs

```diff
@@ -1,4 +1,14 @@
 # bedrock-https-agent
 
+## Config
+
+
+```js
+const {config} = require('bedrock');
+
+// set this to false to ignore SSL errors in development.
```

Comment:
In many READMEs we've just linked to `lib/config.js` and left comments there.  May need a related comment there in this case too.

Reply:
I added a comment to the config file: https://github.com/digitalbazaar/bedrock-https-agent/pull/2/commits/f1c759f03f7b0a979b82f0f4c0beebb0a8bb69e4